Configurations/10-main.conf

```diff
@@ -243,12 +243,12 @@ sub vms_info {
                                               release => "-xO5 -xdepend -xbuiltin"),
                                        threads("-D_REENTRANT")),
         thread_scheme    => "pthreads",
-        lflags           => add("-xarch=generic64",threads("-mt")),
+        lflags           => add(threads("-mt")),
```

Comment:
Oh! So in other words lflags are also always used with cflags. Fair enough...

Reply:
Yup. If you look at Makefile.shared before this change, you will see that CFLAGS and LDFLAGS were always used when linking apps. Those are populated from cflags and lflags respectively (sometim